### ISSUE TITLE
Encounter's posted on OpenMRS does not post location_id

### ISSUE THREAD

**Issue by prithi69** [Intelehealth/intelehealth-openmrs-module#45]
+--------------+------------+--------+-------------+
| encounter_id | concept_id | obs_id | location_id |
+--------------+------------+--------+-------------+
|          520 |     163202 |   1440 |        NULL |
|          520 |     163206 |   1441 |        NULL |
|          520 |     163205 |   1442 |        NULL |
|          520 |     162169 |   1443 |        NULL |
|          518 |       5092 |   1454 |          11 |
|          518 |       5086 |   1470 |          11 |
|          519 |     163212 |   1471 |          11 |
|          518 |       5088 |   1479 |          11 |
|          519 |     163210 |   1480 |          11 |
|          519 |     163211 |   1481 |          11 |
|          519 |     163213 |   1482 |          11 |
|          518 |       5090 |   1483 |          11 |
|          518 |       5089 |   1484 |          11 |
|          518 |       5087 |   1485 |          11 |
|          518 |       5085 |   1486 |          11 |
+--------------+------------+--------+-------------+
15 rows in set (0.00 sec)

Here the location_id is Null..of encounter type visit note.. 
Location_id   Location_name
             2     | Remote                |
|          11      | Telemedicine clinic 1 |
|          10      | Telemedicine clinic 2

**Comment by nehagoel2** [comment]
For the short term we can understand locations through the Geo tags

**Comment by amalafrozalam** [comment]
@nehagoel2 or @prithi69 is this an Android thing or an OpenMRS module thing?

**Comment by hardikk08** [comment]
I have included location along with the encounter but this is what is happening, the location of patient on active visits page is showing the location of encounter type visit note. It is supposed to show nothing if a patient has been seen! 

![screenshot 34](https://user-images.githubusercontent.com/29514954/39123142-8df8c29e-4714-11e8-84ce-9afeb612d87f.png)


Most probably because of this particular code, I tried removing latest and if statements too but with no luck.

![image](https://user-images.githubusercontent.com/29514954/39122935-f625ff2c-4713-11e8-8309-e6e971e29456.png)
